lua/cmake-tools/config.lua

```diff
@@ -18,6 +18,8 @@ local Config = {
   kit = nil,
   configure_preset = nil,
   build_preset = nil,
+  executor = nil,
+  terminal = nil,
```

Comment:
There should be executor and runner in the future. Now it is a little confusing.

Reply:
But for this PR, it's okay.